code/obj/item/device/radios/intercoms.dm

```diff
@@ -26,15 +42,9 @@
 		screen_image.alpha = 180
 		src.UpdateOverlays(screen_image, "screen")
 		if(src.pixel_x == 0 && src.pixel_y == 0)
-			switch(src.dir)
-				if(NORTH)
-					pixel_y = -21
-				if(SOUTH)
-					pixel_y = 24
-				if(EAST)
-					pixel_x = -21
-				if(WEST)
-					pixel_x = 21
+			change_dir(src,null,src.dir)
+	mats = 3
```

Comment:
Any particular reason why this is under `New()` and not just the intercom definition?

Reply:
No I just kept forgetting to move that